worlds/pokemon_crystal/options.py

```diff
@@ -0,0 +1,374 @@
+    }
+
+
+class RandomizeBaseStats(Choice):
+    """vanilla: Vanilla base stats
+    keep_bst: Random base stats, but base stat total is preserved
+    completely_random: Base stats and BST are completely random"""
+    display_name = "Randomize Base Stats"
+    default = 0
+    option_vanilla = 0
+    option_keep_bst = 1
+    option_completely_random = 2
+
+
+class RandomizeTypes(Choice):
+    """vanilla: Vanilla Pokemon types
+    follow_evolutions: Types are randomized but preserved when evolved
+    completely_random: Types are completely random"""
+    display_name = "Randomize Types"
+    default = 0
+    option_vanilla = 0
+    option_follow_evolutions = 1
+    option_completely_random = 2
+
+
+class RandomizePalettes(Choice):
+    """vanilla: Vanilla Pokemon color palettes
+    match_types: Color palettes match Pokemon Type
+    completely_random: Color palettes are completely random"""
+    display_name = "Randomize Palettes"
+    default = 0
+    option_vanilla = 0
+    option_match_types = 1
+    option_completely_random = 2
+
+
+class RandomizeMusic(Toggle):
+    """Randomize all music"""
+    display_name = "Randomize Music"
+    default = 0
+
+
+class RandomizeSFX(Toggle):
+    """Randomize all sound effects"""
+    display_name = "Randomize SFC"
+    default = 0
+
+
+class FreeFlyLocation(Choice):
+    """If enabled, unlocks a random fly location for free.
+    If free_fly_and_map_card is selected, an extra fly location
+    is unlocked when the pokegear and map card are obtained"""
+    display_name = "Free Fly Location"
+    default = 0
+    option_off = 0
+    option_free_fly = 1
+    option_free_fly_and_map_card = 2
+
+
+class HMBadgeRequirements(Choice):
+    """vanilla: HMs require their vanilla badges
+    no_badges: HMs do not require a badge to use
+    add_kanto: HMs can be used with the Johto or Kanto badge"""
+    display_name = "HM Badge Requirements"
+    default = 0
+    option_vanilla = 0
+    option_no_badges = 1
+    option_add_kanto = 2
+
+
+class ReusableTMs(Toggle):
+    """TMs can be used an infinite number of times"""
+    display_name = "Reusable TMs"
+    default = 0
+
+
+class GuaranteedCatch(Toggle):
+    """Balls have a 100% success rate"""
+    display_name = "Guaranteed Catch"
+    default = 0
+
+
+class MinimumCatchRate(Range):
+    """Sets a minimum catch rate for wild Pokemon"""
+    display_name = "Minimum Catch Rate"
+    default = 0
+    range_start = 0
+    range_end = 255
+
+
+class BlindTrainers(Toggle):
+    """Trainers have no vision and will not start battles unless interacted with"""
+    display_name = "Blind Trainers"
+    default = 0
+
+
+class BetterMarts(Toggle):
+    """Improves the selection of items at Pokemarts"""
+    display_name = "Better Marts"
+    default = 0
+
+
+class ExpModifier(Range):
+    """Scale the amount of Experience Points given in battle.
+    Default is 20, for double set to 40, for half set to 10, etc.
+    Must be between 1 and 255"""
```

Comment:
This line isn't needed because it's clear on the site and in a template
```suggestion
"""
```